Jenkinsfile

```diff
@@ -1,54 +1,64 @@
 pipeline {
-    parameters {
-       choice(name: 'env', choices: ['test', 'prod'], description: 'Pick environment')
-    }
 
     agent {
       dockerfile true
     }
 
     environment {
         APIGEE_CREDS = credentials('apigee')
-        APIGEE_DEVPORTAL_CREDS = credentials('apigee-devportal')
         HOME = '.'
+        APIGEE_ORG = 'strebel-eval'
     }
 
     stages {
         stage('Clean') {
             steps {
               script{
                 if (env.GIT_BRANCH == "master") {
-                    env.APIGEE_PREFIX = ""
+                    env.APIGEE_DEPLOYMENT_SUFFIX = ""
                     env.APIGEE_PROFILE = "test"
+                    env.NORTH_BOUND_API = "strebel-eval-test.apigee.net"
                 } else if (env.GIT_BRANCH == "prod") {
-                    env.APIGEE_PREFIX = ""
+                    env.APIGEE_DEPLOYMENT_SUFFIX = ""
                     env.APIGEE_PROFILE = "prod"
+                    env.NORTH_BOUND_API = "strebel-eval-prod.apigee.net"
                 } else { //feature branches
-                    env.APIGEE_PREFIX = "jenkins"
+                    env.APIGEE_DEPLOYMENT_SUFFIX = env.GIT_BRANCH ? "-" + env.GIT_BRANCH : "-jenkins"
                     env.APIGEE_PROFILE = "test"
+                    env.NORTH_BOUND_API = "strebel-eval-test.apigee.net"
                 }
               }
               sh "mvn clean"
             }
         }
         stage('Static Code Analysis, Unit Test and Coverage') {
             steps {
-              sh "mvn -ntp test -P${env.APIGEE_PROFILE} -Ddeployment.suffix=${env.APIGEE_PREFIX} -Dcommit=${env.GIT_COMMIT} -Dbranch=${env.GIT_BRANCH} -Duser.name=jenkins"
+              script {
+                  AUTHOR_EMAIL = sh (
```

Comment:
@danistrebel - not sure why you are putting the committer as the user.name ?

Reply:
@ssvaidyanathan the idea was to show the committer in the apigee api proxy description as shown below:

![image](https://user-images.githubusercontent.com/1324391/81375215-51b48000-9101-11ea-9993-57cd722777b0.png)

Reply:
Are you referring to committer or deployer? I don't see the committer above

Reply:
They are the same in this case. The commit triggers the deployment on Apigee side. So from the Apigee perspective the committer acts as the deployer. Feel free to adapt the wording in the Apigee comment if you like.